### DISCUSSION TITLE
Model not refreshing

### DISCUSSION THREAD

**Discussion by KostaDokosic** [google/model-viewer#2734] [Q&A]
So when I move the model or enable the auto-rotate option 3d model is not refreshing and removing the last position. It causing a glitch effect.
This is how it looks like: https://ibb.co/3ywQ8np
https://ibb.co/gZMWtgg

and this is how model should look like: https://ibb.co/QcrTv1s

This bug is only happening on cef, not on the actual browser-chrome.

**Comment by elalish** [comment]
What is "cef"? Can you give steps to reproduce the behavior you're seeing?

**Comment by KostaDokosic** [comment]
Chromium embedded framework. I just made a simple setup (index.html) and add 3d model .glb format. On browser, model looks good but on cef object is not refreshing.

**Reply by elalish** [reply]
Sounds like a CEF bug to me. Have you filed an issue there? Would also be helpful to know if a simpler WebGL-style example works. Have you tried any of the three.js [examples](https://threejs.org/examples/?q=gltf#webgl_loader_gltf)?

**Reply by KostaDokosic** [reply]
Well, looks I found what makes a problem. If I enable accelerated cef rendering (hardware rendering) everything works fine. Is there any other way to display 3d model without using this option?

**Reply by elalish** [reply]
Again, everything *should* work with the software renderer as well (though it'll be slower, obviously). In fact we use this mode for our automated testing. I'd still say it sounds like a CEF bug.
